packages/core/admin/admin/src/pages/tests/InternalErrorPage.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { lightTheme, ThemeProvider } from '@strapi/design-system';
+import { render as renderRTL, screen } from '@testing-library/react';
+import { createMemoryHistory } from 'history';
+import { IntlProvider } from 'react-intl';
+import { Router } from 'react-router-dom';
+
+import { InternalErrorPage } from '../InternalErrorPage';
+
+const render = (props) =>
+  renderRTL(<InternalErrorPage {...props} />, {
+    wrapper({ children }) {
+      const history = createMemoryHistory();
+
+      return (
+        <ThemeProvider theme={lightTheme}>
+          <IntlProvider locale="en" messages={{}} textComponent="span">
+            <Router history={history}>{children}</Router>
+          </IntlProvider>
+        </ThemeProvider>
+      );
```

Comment:
This should just use `MemoryRouter` and not provide a `history`, same for `NotFoundPage`